scss/_patterns_strip.scss

```diff
@@ -1,15 +1,17 @@
 // Full width strip pattern
 @mixin vf-p-strip {
 
-  width: 100%;
-  clear: both;
-  background-color: $color-light;
+  .strip {
```

Comment:
Looks like this change is also included in the breadcrumbs PR. With a few comments, please resolve them before landing this branch.